EMCAL/EMCALbase/AliEMCALTriggerSTUDCSConfig.h

```diff
@@ -86,9 +86,6 @@ class AliEMCALTriggerSTUDCSConfig : public TObject
   void    GetSegmentation(TVector2& v1, TVector2& v2, TVector2& v3, TVector2& v4) const;
   
   TClonesArray *GetErrorCountsForTRU(Int_t itru) const;
-  
-protected:
```

Comment:
Indeed the issue seems to be connected with a missing implementation of the operator= (which becomes declared now moving it from protected to public). This can be implemented similarly to the copy constructor.